dataactcore/utils/jsonResponse.py

```diff
@@ -41,10 +47,10 @@ def error(exception, errorCode, **kwargs):
             responseDict[key] = kwargs[key]
 
 
-        exc_type, exc_obj, exc_tb = sys.exc_info()
+        _, _, exc_tb = sys.exc_info()
```

Comment:
Good to know. I'll try it out; the `exc_info` stuff is the definition of an incantation.